src/components/analyses/details/InfoPanel.js

```diff
@@ -142,6 +143,8 @@ function Updates(props) {
                 columnData={columns}
                 rowsInPage={updates.length}
                 baseId={baseId}
+                selectAllLabel={i18nCommon("selectAllLabel")}
+                sortLabel={i18nCommon("sortLabel")}
             />
```

Comment:
May be we could have another component in `Sonora` that will pass `selectAllLabel` and `sortLabel` prop by default to `EnhancedTableHead`.

Reply:
That would make sense to me, to save some repetition. I imagine it'd pretty much just be `export default (props) => <EnhancedTableHead selectAllLabel={t("selectAllLabel")} sortLabel={t("sortLabel")} {...props} />`, easily thrown into a file somewhere or other.